packages/ethereum/deploy/16_HoprNetworkRegistryProxy.ts

```diff
@@ -14,7 +14,8 @@ const main: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const stakeAddress = (await deployments.get('HoprStake')).address
 
   // Local development environment uses HoprDummyProxyForNetworkRegistry. All the other network uses HoprStakingProxyForNetworkRegistry
-  const registryProxyName = network.name == 'hardhat' ? DUMMY_PROXY : STAKING_PROXY
+  // FIXME: Before Dev NFTs are minted in production environment, dummy proxy gets deployed in production
+  const registryProxyName = network.name == 'hardhat' || network.tags.production ? DUMMY_PROXY : STAKING_PROXY
```

Comment:
Does the comment apply to Valencia?

Reply:
Good point, this line needs to be same as `master`, fixing